src/WebJobs.Extensions.DurableTask/Listener/OutOfProcOrchestrationShim.cs

```diff
@@ -127,6 +127,13 @@ private async Task ProcessAsyncActions(AsyncAction[][] actions)
                         case AsyncActionType.CreateTimer:
                             using (var cts = new CancellationTokenSource())
                             {
+                                DurableOrchestrationContext ctx = this.context as DurableOrchestrationContext;
+
+                                if (ctx != null)
+                                {
+                                    tasks.Add(ctx.OutOfProcCreateTimer(ctx, action.FireAt, cts.Token));
+                                }
```

Comment:
Shouldn't line 137 be in an else case of this?

Reply:
Yeah it should, just fixed it.